### ISSUE TITLE
Dataset preparation

### ISSUE THREAD

**Issue by Naxalov** [Naxalov/PyTorch_VGG#2]
- [ ] Loading a Dataset
- [ ] Visualizing the Dataset
- [ ] Split dataset to train and test
- [ ] Preparing  data for training with DataLoaders
- [ ] Display image and label.
